src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+import core.basesyntax.service.ColorSupplier;
+
+public class Main {
+    public static void main(String[] args) {
+        ColorSupplier colorSupplier = new ColorSupplier();
+        FigureSupplier figureSupplier = new FigureSupplier(colorSupplier);
+
+        AbstractFigure[] figures = new AbstractFigure[3];
+
+        for (int i = 0; i < 3; i++) {
+            figures[i] = figureSupplier.getRandomFigure();
+        }
+
+        for (AbstractFigure figure : figures) {
+            figure.draw();
+        }
+    }
```

Comment:
read the task description again
don't hardcode lengths 2+ times, use figures.length
you need only one for loop in this class
Half of figures should be random, the other half - default